lib/engine/game/g_18_ireland/round/stock.rb

```diff
@@ -17,6 +17,10 @@ def finish_round
             end
             super
           end
+
+          def corporations_to_move_price
```

Comment:
Default for minors changed to not float at the end of the stock round. 18Ireland is special in that minors do float, so put the special case here instead of in the mainline code.